db/migrate/20201027134205_add_immersible_to_organization.rb

```diff
@@ -0,0 +1,5 @@
+class AddImmersibleToOrganization < ActiveRecord::Migration[5.1]
+  def change
+    add_column :organizations, :immersible, :boolean, default: false
```

Comment:
Because of migration performance reasons, we tend to avoid to add default boolean values, at least in big tables like users and assignments. But it is ok to add it for organizations, right @luchotc ?

Reply:
I removed the default, just for consistency with the other fields